dev/plots_api_strawman.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/python
+
+# ======================================
+# Plots for Observation Planning Toolbox
+# ======================================
+
+# This is a strawman API for plotting with Astroplan.
+
+# Must first construct an Observer object...
+
+from astroplan import Observer
+import astropy.units as u
+import pytz
+
+obs = Observer(name='Subaru Telescope',
+               longitude='-155:28:48.900',
+               latitude='+19:49:42.600',
+               elevation=4163 * u.meter,
+               pressure=0.615 * u.bar,
+               relative_humidity=0.11,
+               temperature=0 * u.deg_C,
+               timezone=pytz.timezone('US/Hawaii'),
+               description="Subaru Telescope on Mauna Kea, Hawaii")
+
+day = obs.get_date("2015-05-01")
+
+# ...then define your target... 
+
+from astroplan import FixedTarget
+
+t1 = FixedTarget(name='Polaris', ra='02:31:49.09', dec='+89:15:50.8')
+
+# ...once you've constructed your Observer and Target objects, you can plot any
+#   any number of quantities.
+
+
+# ===============
+# Airmass vs Time
+# ===============
+
+from astroplan import plot_airmass
+
+# The default airmass plot takes a Target, an Observer, and a datetime object
+#   as arguments. 
+# The default time range is six hours before to six hours after the datetime
+#   specified, and a light background.
+plot_airmasss(t1, observer=obs, datetime=day)
+
+# You can also pass several targets to plot_airmass once you create them.
+
+target_list = [t1, t2, t3]
+
+plot_airmass(target_list, observer=obs, datetime=day)
+
+# You can set several options, such as change the time start/end (relative
+#   to the datetime object), set a dark background, etc., with optional keywords.
+
+plot_airmass(target_list, observer=obs, datetime=day, time_window=(-8, +8), dark)
+
+# You can leave the start or end time to the default simply by leaving that 
+#   option blank.
+
+plot_airmass(target_list, observer=obs, datetime=day, time_window=(-8,), dark)
```

Comment:
This will fail because dark is a non-keyword argument passed in after keyword arugments -- bu what do you imagine this variable `dark` is? Perhaps we want to instead allow for setting a global "dark mode" as well as a keyword argument (maybe `dark_mode=True`)?